doc/man3/EVP_PKEY_get_size.pod

```diff
@@ -45,11 +47,52 @@ EVP_PKEY_get_security_bits() returns the number of security bits of the given
 I<pkey>, bits of security is defined in NIST SP800-57.
 This corresponds to the provider parameter B<OSSL_PKEY_PARAM_SECURITY_BITS>.
 
+EVP_PKEY_get_security_category() returns the
+L<post-quantum security category|https://nvlpubs.nist.gov/nistpubs/ir/2024/NIST.IR.8547.ipd.pdf>
+as defined by NIST.
+The post-quantum security category is an integer value from 0 to 5 that
+is based an algorithm's classification on the range of security strengths
```

Comment:
an = on a

Reply:
I don't understand what's being asked here.  "on the" is correct, "on a" would be wrong since it is talking about a specific categorisation.

Reply:
Look at the next an in the sentence and read it :)